2020-01-17 16:22:13 INFO  Operation directory: /Users/johnwalker/projects/baseball/WBBaseballStats/data/Scripts/logs/LOAD_20200117-162213-545130
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.36.65:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.36.65:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.36.65:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.36.65:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.36.65:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.36.65:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.34.40:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.34.40:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.34.40:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.34.40:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.34.40:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.34.40:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.36.65:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.36.65:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.36.65:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.36.65:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.36.65:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.36.65:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.36.65:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.36.65:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.36.65:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.34.40:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.34.40:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.34.40:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.36.65:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.36.65:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.36.65:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.34.40:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.34.40:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.34.40:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.34.40:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.34.40:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.34.40:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.34.40:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.34.40:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.34.40:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.34.40:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.34.40:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.34.40:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.34.40:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.34.40:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.34.40:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating pool to /172.31.34.40:9042
com.datastax.driver.core.exceptions.ConnectionException: [/172.31.34.40:9042] Pool was closed during initialization
	at com.datastax.driver.core.HostConnectionPool$2.onSuccess(HostConnectionPool.java:164)
	at com.datastax.driver.core.HostConnectionPool$2.onSuccess(HostConnectionPool.java:149)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:213) [34 skipped]
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.36.65:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.36.65:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.36.65:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating connection to /172.31.36.65:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.36.65:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.36.65:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-17 16:22:20 WARN  Error creating pool to /172.31.36.65:9042
com.datastax.driver.core.exceptions.ConnectionException: [/172.31.36.65:9042] Pool was closed during initialization
	at com.datastax.driver.core.HostConnectionPool$2.onSuccess(HostConnectionPool.java:164)
	at com.datastax.driver.core.HostConnectionPool$2.onSuccess(HostConnectionPool.java:149)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:213) [34 skipped]
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
2020-01-17 16:22:58 INFO  Operation LOAD_20200117-162213-545130 completed successfully in 38 seconds.
2020-01-17 16:23:00 INFO  Final stats:
2020-01-17 16:23:00 INFO  Records: total: 104,324, successful: 104,324, failed: 0
2020-01-17 16:23:00 INFO  Batches: total: 96,631, size: 1.07 mean, 1 min, 5 max
2020-01-17 16:23:00 INFO  Memory usage: used: 141 MB, free: 158 MB, allocated: 299 MB, available: 3,641 MB, total gc count: 16, total gc time: 129 ms
2020-01-17 16:23:00 INFO  Writes: total: 104,324, successful: 104,324, failed: 0, in-flight: 0
2020-01-17 16:23:00 INFO  Throughput: 2,563 writes/second
2020-01-17 16:23:00 INFO  Latencies: mean 183.16, 75p 238.03, 99p 771.75, 999p 1,434.45 milliseconds
2020-01-17 16:23:00 INFO  Last processed positions can be found in positions.txt
